src/util/segmentation/operations/eraseInsideBoundingBox.js

```diff
@@ -0,0 +1,37 @@
+import { getBoundingBoxAroundPolygon } from '../boundaries';
+
+/**
+ * EraseInsideBoundingBox - Erase all pixels labeled with the activeSegmentIndex,
+ * in the bouding box containing all points from evt.operationData.points.
+ * @param  {} evt The Cornerstone event.
+ * @param {} evt.operationData An object containing the `pixelData` to
+ *                          modify, the `segmentIndex` and the `points` array.
+ * @returns {null}
+ */
+export default function eraseInsideBoundingBox(evt) {
```

Comment:
Basically `eraseInside.js`, but without the pointInPolygon filter 🤔

Reply:
This is reused. `eraseInside.js` was badly named, thats a strategy for freehandscissors. We should probably stick all the helpers in a subdirectory.